### ISSUE TITLE
Locally defined mixins with "-" (minus) in the name are not propery resolved

### ISSUE THREAD

**Issue by r00t3g** [tomblachut/svelte-intellij#149]
Not sure if it is a problem with Svelte plugin, however there seems to be a problem in resolving locally defined mixins in `<style lang="scss">` tags when there is a minus sign in their names. Imported from mixins are OK regardless of their naming.

![image](https://user-images.githubusercontent.com/579779/90602199-0894c180-e202-11ea-9855-f976bc39cf65.png)

Reproduction code:

```html
<style lang="scss" type="text/scss">
    @import 'mixins.scss'; // external-test-mixin defined here
    
    @mixin test() {
        opacity: 1;
    }
    
    @mixin test-mixin() {
        opacity: 1;
    }
    
    .my-rule {
        @include test;
        @include external-test-mixin;
        @include test-mixin;
    }
</style>
```

**Comment by r00t3g** [comment]
In regular HTML files the problem does not occur.

![image](https://user-images.githubusercontent.com/579779/90602765-ee0f1800-e202-11ea-99fa-1b7f4931d990.png)

**Comment by tomblachut** [comment]
whole CSS resolving is not complete, need to gather everything later and figure out solution. Thx for reporting

**Comment by tomblachut** [comment]
Fix released as part of 0.17.*
